app/views/hyrax/homepage/_recently_uploaded.html.erb

```diff
@@ -23,7 +23,7 @@
   <ul class="list-inline datasets-highlights-list">
     <li>
       <%= link_to "Browse datasets",
-          main_app.search_catalog_path(f: { has_model_ssim: ["Dataset"], top_level_bsi: ["True"] }),
+          main_app.search_catalog_path(f: { has_model_ssim: ["Dataset"], Rdr::Index::Fields.top_level => ["True"] }),
```

Comment:
Should this be: 

`Rdr::Index::Fields.top_level => "True"`

i.e., string, not array?